public/index.html

```diff
@@ -73,6 +73,7 @@
     <main id="main-container"></main>
     <script src="vendor/d3-6.3.1/d3.min.js"></script>
     <script src="vendor/d3-legend-2.25.6/d3-legend.min.js"></script>
+    <script src="https://unpkg.com/d3-simple-slider"></script>
```

Comment:
we should host this locally under vendor/

Reply:
Updated, thanks.